### ISSUE TITLE
oraclejdk10 broken

### ISSUE THREAD

**Issue by IRus** [sormuras/bach#37]
See builds:
- https://travis-ci.com/Heapy/tgto/builds/88304955
- https://travis-ci.org/bootique/bootique-io

**Comment by sormuras** [comment]
The URL used in the script seems to be valid: http://download.oracle.com/otn-pub/java/jdk/10.0.2+13/19aef61b38124481863b1413dce1855f/jdk-10.0.2_linux-x64_bin.tar.gz

`install-jdk.sh` will focus on supporting **OpenJDK** builds, especially the EA builds. Perhaps you may switch to OpenJDK 11 and/or turn off the job that depends on Oracle JDK 10?

**Comment by IRus** [comment]
Yes, I already did this for my projects, but such situations abnormal. Do you know why such errors actually happens?

**Comment by sormuras** [comment]
Further information on the topic: https://travis-ci.community/t/help-with-oraclejdk10-build-on-linux/398

**Comment by IRus** [comment]
Thing that works yesterday, should work tomorrow, especially on CI, especially in script that official on such thing like travis-ci. There are should be some storage level, something like IPFS, to cache all old images and din't broke build because of third-party.
